Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -267,13 +267,18 @@ class PlaceDetailsCardView: UIView {
             return ("No hours", "For Today") // TODO: probably closed today - how best to handle?
         }
 
-        if date > open {
+        // It's icky: stored times are normalized (e.g. date is always 2000-01-01) so we need to
+        // normalize the argument date to compare them. However, this date should not be used to
+        // retrieve the strings because the date is used in those cases to get the day of the
+        // week, for which the date can't be normalized.
+        let comparisonDate = OpenHours.getNormalizedDate(date)
+        if comparisonDate > open {
             let closeTimeStr = openHours.getCloseTimeString(forDate: date)
-            if date < close {
+            if comparisonDate < close {
                 return (closeTimeStr, "Closing time")
             }
             print("lol venue unexpectedly already closed for \(date) and closing \(close)")
-            return ("Closed", "at \(closeTimeStr)") // TODO: already closed - how best to handle?
+            return ("Already closed", "at \(closeTimeStr)") // TODO: already closed - how best to handle?
```

Comment:
I am not sure what conversation's you've had with UX about this, but I would say:
* If the place is not yet open today, display the opening time
* If the place is open today, display the closing time
* If the place is closed, we should display the opening time for tomorrow